backend/routes/policies.js

```diff
@@ -0,0 +1,72 @@
+const { Policies } = require('../models')
+const { authShop } = require('./_auth')
+
+/*
+Use the [2] lines below to enable logging to the Terminal
+const { getLogger } = require('../utils/logger')
+const log = getLogger('routes.policies')
+*/
+
+/*
+References:
+https://sequelize.org/master/manual/model-querying-basics.html
+https://expressjs.com/en/guide/routing.html
+*/
+
+module.exports = function (router) {
+  // Apply the callback 'authShop' to any POST request to the endpoint '/shop/policies'
```

Comment:
nit: in express jargon, authShop is a `middleware`, not a `callback`

Reply:
Updated.